source/security/index.html.md.erb

```diff
@@ -27,13 +27,13 @@ source code repositories.
 
 Follow these steps to revoke your API key immediately if you believe it has been accidentally shared or compromised:
 
-1. Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login)
+1. Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login).
 
-2. Select the correct services in the [My services](https://selfservice.payments.service.gov.uk/my-services) section
+2. Select the correct services in the [My services](https://selfservice.payments.service.gov.uk/my-services) section.
 
-3. Navigate to the [API keys](https://selfservice.payments.service.gov.uk/api-keys) page in the Settings section.
+3. Select **API keys**.
```

Comment:
Hi @kbottla - as mentioned in the PR description, it's good practice to avoid describing every UI step where possible. It's intuitive to go from 'Settings' to 'API keys', so we should be good not to go in that level of detail here. Hope that's ok.